bcda/cli.go

```diff
@@ -0,0 +1,135 @@
+package main
```

Comment:
I think this should be cclf.go maybe?

Reply:
I was thinking we could eventually move the other CLI functions-- basically everything after `setUpApp()`-- from main.go to cli.go. Or do you think it makes more sense to have them split into separate files based on what they do?

Reply:
That is fine with me; I just wasn't sure.